x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/add_source_logic.test.ts

```diff
@@ -93,75 +93,124 @@ describe('AddSourceLogic', () => {
     expect(AddSourceLogic.values).toEqual(defaultValues);
   });
 
+  const setupExpectedState = ({ setSourceConfigData = {} } = {}) => {
```

Comment:
Why did you call it `setExpectedState` in the previous commit but here `setupExpectedState`?